onnxruntime/core/providers/xnnpack/detail/node_support_checker.cc

```diff
@@ -35,6 +38,7 @@ const Node* ClipReluChecker(const Node& node,
                             const GraphViewer& graph,
                             const std::unordered_set<const Node*>& supported_nodes) {
   const Node* fuse_with{nullptr};
+  static const std::unordered_set<std::string> node_to_be_fuse = {"Conv", "MaxPool", "AveragePool"};
```

Comment:
There are two reason I didn't fuse Qlinearnode or any QDQGroup operators with relu/clip here.
- It would make the fusion tricky based on current design, as QDQGroup fusion and node-fusion are all happended in the second call of GetCapability.
- QlinearConv will be clipped betweem 0-255 or -127-127 automatically, so relu/clip/relu6 will be fused in quantization process.
Please correct me if I was wrong.
So I have a check in the fusion process.
- the activation node must not be QDQgroup
- the input node must nobe be QDQgroup

Reply:
Yes. I need to understand what's the case of a QDQ group fusion with anohter QDQ group. How would that appear in models.